tests/components/knx/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Tests for the KNX integration."""
```

Comment:
This is a workaround, we should not do this until there are actually tests

Reply:
First tests are on their way anyway. https://github.com/home-assistant/core/pull/48311
I just wanted to avoid awkward conflicts for other PRs that break mypy checks without getting notified (because of `ignore_missing_imports = True`). 
I'll remove it again...

Reply:
Well, easy solution: Merge #48311 first, rebase this one.

Reply:
Sure, go ahead.

Reply:
So tests are available now. I think we are set.